plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTypeMapping.java

```diff
@@ -0,0 +1,596 @@
+                .addRoundTrip(
+                        "map<text, frozen<address>>",
+                        "" +
+                                "{" +
+                                "    'home': {" +
+                                "                street: '1600 Pennsylvania Ave NW'," +
+                                "                city: 'Washington'," +
+                                "                zip: '20500'," +
+                                "                phones: {" +
+                                "                            'cell': {" +
+                                "                                        country_code: 1," +
+                                "                                        number: '202 456-1111'" +
+                                "                                    }," +
+                                "                            'landline': {" +
+                                "                                            country_code: 1," +
+                                "                                            number: '...'" +
+                                "                                        }" +
+                                "                        }" +
+                                "            }," +
+                                "    'work': {" +
+                                "                street: '1600 Pennsylvania Ave NW'," +
+                                "                city: 'Washington'," +
+                                "                zip: '20500'," +
+                                "                phones: {" +
+                                "                            'fax': {" +
+                                "                                       country_code: 1," +
+                                "                                       number: '...'" +
+                                "                                   }" +
+                                "                        }" +
+                                "            }" +
+                                "}",
+                        VARCHAR,
+                        "CAST(" +
+                                "'{" +
+                                "\"home\":\"{" +
+                                "street:''1600 Pennsylvania Ave NW''," +
+                                "city:''Washington'',zip:''20500''," +
+                                "phones:{''cell'':{country_code:1,number:''202 456-1111''},''landline'':{country_code:1,number:''...''}}}\"," +
+                                "\"work\":\"{" +
+                                "street:''1600 Pennsylvania Ave NW''," +
+                                "city:''Washington''," +
+                                "zip:''20500''," +
+                                "phones:{''fax'':{country_code:1,number:''...''}}}\"" +
+                                "}' AS VARCHAR)")
```

Comment:
@ebyhr When I format the code, the code looks like this, which looks unreadable, do you have any suggestions?

Reply:
I think we can use more simple table definition - one column and non-nested UDT.

Reply:
Okay, I'll fix this, this test case is actually an example from the [Cassandra documentation](https://cassandra.apache.org/doc/latest/cassandra/cql/types.html#udts).